ciphers/base85.py

```diff
@@ -1,30 +1,57 @@
-import base64
+"""
+Base85 (Ascii85) encoding and decoding
 
+https://en.wikipedia.org/wiki/Ascii85
+"""
 
-def base85_encode(string: str) -> bytes:
+
+def _base10_to_85(d: int) -> str:
+    return "".join(chr(d % 85 + 33)) + _base10_to_85(d // 85) if d > 0 else ""
+
+
+def _base85_to_10(digits: list) -> int:
+    return sum(char * 85**i for i, char in enumerate(digits[::-1]))
+
+
+def ascii85_encode(data: bytes) -> bytes:
     """
-    >>> base85_encode("")
+    >>> ascii85_encode(b"")
     b''
-    >>> base85_encode("12345")
+    >>> ascii85_encode(b"12345")
     b'0etOA2#'
-    >>> base85_encode("base 85")
+    >>> ascii85_encode(b"base 85")
     b'@UX=h+?24'
     """
-    # encoded the input to a bytes-like object and then a85encode that
-    return base64.a85encode(string.encode("utf-8"))
+    binary_data = "".join([bin(ord(d))[2:].zfill(8) for d in data.decode("utf-8")])
```

Comment:
Can we use a generator instead?
```suggestion
    binary_data = "".join(bin(ord(d))[2:].zfill(8) for d in data.decode("utf-8"))
```